.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
@EoinShaughnessy Just flagging this particular one for your eyes. I know it's the same one we use in the Design System repo, just thought you'd like to have a look since this repo is more documentation focused and we might possibly want to do something different here.